### PR TITLE
Feature/64/create what you can do section

### PR DESCRIPTION
**Pull request by DeLettenhove**
Create what can you do section on the guest home page.

image

### CODE REVIEW

File: src/components/card/Card.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Box, Button, Typography } from '@mui/material'
+
+import { cardStyles as style } from '~/components/card/card.styles'
+import { Link } from 'react-router-dom'
+
+//TODO add possibility for button to redirect user to the specified url.
+
+const Card = ({ img, title, description, btnText, link }) => {
```
Comment (tarasSanotskyi) [inline src/components/card/Card.js:8]: give a more specific name to the component

File: src/containers/guest-home-page/WhatCanYouDo.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        description={ t(item.description) }
+        img={ item.img }
+        key={ item.id }
+        link={ item.link }
+        title={ t(item.title) }
+      />)
+  })
+
+  return (
+    <Box sx={ style.container }>
```
Comment (tarasSanotskyi) [inline src/containers/guest-home-page/WhatCanYouDo.js:44]: add id to this component

File: src/components/card/Card.js (hunk 1 above)
Comment (DeLettenhove) [reply src/components/card/Card.js]: Fixed.

File: src/components/info-card/info-card.styles.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    boxSizing: 'border-box',
+    maxWidth: {
+      md: '427px',
+      xs: '343px',
+    },
+    padding: '24px 24px 32px',
+    backgroundColor: theme.palette.basic.white,
+    boxShadow: theme.shadows.primary,
+    borderRadius: '6px',
+    '&:not(:last-child)': {
```
Comment (DeLettenhove) [inline src/components/info-card/info-card.styles.js:18]: I use this pseudo classes for adding indent between cards. It's fine or i should look for alternate solution?

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/components/info-card/info-card.styles.js (hunk 3 above)
Comment (DeLettenhove) [reply src/components/info-card/info-card.styles.js:18]: Fixed.

File: src/components/heading/Heading.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { Typography } from '@mui/material'
+import { headingStyles as style } from '~/components/heading/heading.styles'
+
+const Heading = ({ title, description }) => {
+  return (
+    <>
```
[the remark this answers is not in the document]
Comment (DeLettenhove) [reply src/components/heading/Heading.js:6]: Fixed.

File: src/components/heading/heading.styles.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+export const headingStyles = {
+  title: {
+    typography: {
+      md: 'h3',
+      xs: 'h4'
+    },
+    marginBottom: '16px',
+  },
+  description: {
+    marginBottom: {
```
[the remark this answers is not in the document]
Comment (DeLettenhove) [reply src/components/heading/heading.styles.js:10]: Fixed.

File: tests/unit/containers/guest-home-page/WelcomeBlock.spec.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
+import { screen, render } from '@testing-library/react'
+import { MemoryRouter } from 'react-router-dom'
+
+import WelcomeBlock from '~/containers/guest-home-page/WelcomeBlock'
+
+describe('WelcomeBlock component', () => {
+  beforeEach(() => {
+    render(
+      <MemoryRouter>
```
[the remark this answers is not in the document]
Comment (DeLettenhove) [reply tests/unit/containers/guest-home-page/WelcomeBlock.spec.js:9]: Fixed.

File: src/components/info-card/InfoCard.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+//TODO add possibility for button to open pop-up.
+
+const InfoCard = ({ img, title, description, actionLabel, cardWidth }) => {
+  const { t } = useTranslation()
+
+  return (
+    <Box sx={ { ...styles.card, maxWidth: cardWidth } }>
+      <Box
+        alt={ t(title) }
```
Comment (maarekua) [inline src/components/info-card/InfoCard.js:14]: Move translations to the parent component.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/components/info-card/InfoCard.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+}
+const descriptionVariant = {
+  md: 'body1',
+  xs: 'body2'
+}
+
+const InfoCard = ({ img, title, description, actionLabel, cardWidth }) => {
+  return (
+    <Box sx={ { ...styles.card, maxWidth: cardWidth } }>
+      <Box
```
Comment (rostyslaw27) [inline src/components/info-card/InfoCard.js:20]: getByAltText() instead of data-testid

File: src/containers/guest-home-page/Welcome.js [hunk 9]
```diff
@@ -20,18 +24,23 @@ const WelcomeBlock = () => {
   return (
     <Box sx={ styles.container }>
       <Box
-        alt='Title' component='img' src={ images[size] }
+        alt='Title'
```
Comment (rostyslaw27) [inline src/containers/guest-home-page/Welcome.js:27]: getByAltText

File: src/containers/guest-home-page/Welcome.js [hunk 10]
```diff
@@ -20,18 +24,23 @@ const WelcomeBlock = () => {
...
+        component='img' data-testid='welcomeTitleImg' src={ images[size] }
         sx={ styles.title }
       />
-      <Typography sx={ styles.subtitle }>
+      <Typography data-testid='welcomeDescription' sx={ styles.subtitle }>
```
Comment (rostyslaw27) [inline src/containers/guest-home-page/Welcome.js:31]: getByText

File: src/containers/guest-home-page/Welcome.js [hunk 11]
```diff
@@ -20,18 +24,23 @@ const WelcomeBlock = () => {
...
         { t('guestHomePage.welcomeBlock.description') }
       </Typography>
 
-      <Button sx={ styles.getStartBtn } variant="contained">
+      <Button
+        component={ Link }
+        data-testid='welcomeGetStarted' sx={ styles.getStartBtn } to={ scrollTo }
```
Comment (rostyslaw27) [inline src/containers/guest-home-page/Welcome.js:37]: getByText

**Review by rostyslaw27 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**